generation/2d_ldm/2d_ldm_tutorial.ipynb

```diff
@@ -0,0 +1,1526 @@
+{
```

Comment:
<p>The same concerns here as with 3D LDM</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>